examples/wagtail/core/jinja/home_page.jinja

```diff
@@ -15,92 +16,92 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>{{ page.title }}</title>
-    <link rel="stylesheet" href="{{ static('css/theme.css') }}">
-    <link rel="stylesheet" href="{{ static('css/styles.css') }}">
-    <link rel="stylesheet" href="{{ static('css/ogcio-ds-0.4.4.min.css')}}">
+	<meta charset="UTF-8">
```

Comment:
so are we doing 4 spaces indentation or 2 ? 😄

Reply:
Just fixing this now :D